simulation/test_torrent_status.cpp

```diff
@@ -41,11 +41,161 @@ using namespace sim;
 namespace lt = libtorrent;
 
 // this is a test for torrent_status time counters are correct
-TORRENT_TEST(status_timers)
```

Comment:
I will try it, at least I have to refactor all the new tests because at somepoint the got messed without my notice. I just didn't understand when this did happen.

I guess whole part of them was only an unfinished draft and after some days I may find them in my repo and did check them in because the did look finish and the local build was sucessfull.
Yesterday I try to finish this branch and found out that nothing does work.
 For example the paused clamp test would never work because on pause there is no seeding.